### DISCUSSION TITLE
Running a language model for spaCy 0.101

### DISCUSSION THREAD

**Discussion by rkrovetz** [explosion/spaCy#7031] [Language Support]
I would like to make a comparison between spaCy 0.101 and the current version.     The older version of spaCy is running on an old computer, and I would like to get it running on my current computer.    I set up a virtual environment for each each version.   But when I try to run spaCy 0.101.0, I get the following results:

$ source spacy0101/bin/activate
$ cd spacy0101
$ pip2 install spacy==0.101.0
$ python
Python 2.7.18 (default, Aug  4 2020, 11:16:42) 
[GCC 9.3.0] on linux2
Type "help", "copyright", "credits" or "license" for more information.
>>> import spacy
>>> from spacy.en import English
>>> import os
>>> nlp = English(parser=False, tagger=True, entity=False)

The end of the Traceback says:
RuntimeError: Model 'en>=1.1.0,<1.2.0' not installed. Please run 'python -m spacy.en.download' to install latest compatible model.

If I try the command to load the latest model, I get an error about the certificate:

ssl.CertificateError: hostname 'index.spacy.io' doesn't match 'briefkasten.linuxkiste.net'

How can I get the old version to run on my current computer?

Thanks,
Bob

**Comment by honnibal** [comment]
Hi Bob,

You should be able to copy the model data out of the `spacy/data` directory within the installed package that's on the old computer. Then you can either set that up in the same place in the new computer, or you can pass the directory path to `spacy.load()` (I think that worked already by 0.101, anyway -- I don't remember!).
